src/site/content/en/blog/trusted-types/index.md

```diff
@@ -260,7 +260,7 @@ Sometimes you can't change the offending code. For example, this is the case if
 ```javascript
 if (window.trustedTypes && trustedTypes.createPolicy) { // Feature testing
   trustedTypes.createPolicy('default', {
-    createHTML: (string, sink) => DOMPurify.sanitize(string, {RETURN_TRUSTED_TYPE: true});
+    createHTML: (string, sink) => DOMPurify.sanitize(string, {RETURN_TRUSTED_TYPE: true})
```

Comment:
While we're here: is `sink` unnecessary?

Reply:
For this snippet, it's not used, but I wanted to show that the sink value would be passed by the API to the function, such that e.g. custom logging, or different behavior per-sink could be implemented.

Reply:
If we removed it, we'd have to explain in text that there's a missing argument. Otherwise, readers might question whether the article was up to date with the spec. That's a reasonable concern given the cutting edge nature of our articles.